src/Catalog/Tests/Api/Rest/CatalogsTest.php

```diff
@@ -15,10 +15,10 @@
 namespace Gally\Catalog\Tests\Api\Rest;
 
 use Gally\Catalog\Model\Catalog;
-use Gally\Test\AbstractEntityTest;
+use Gally\Test\AbstractEntityTestWithUpdate;
```

Comment:
So, does Gally\Test\AbstractEntityTest still have any use if we are replacing it with AbstractEntityTestWithUpdate everywhere ?

Reply:
yes here [api/packages/gally-standard/src/Index/Model/Index.php](https://github.com/Elastic-Suite/gally-standard/blob/6f2c1ab45da95545a88140b6a58fd98a0314c8c6/src/Index/Tests/Api/Rest/IndexOperationsTest.php#L21)

Reply:
Yes but it could also be used there, couldn't it ?
Accepting as it is, but we'll need to discuss that ;)